packages/shared/lib/core/wallet/utils/getActivityTileTitle.ts

```diff
@@ -28,8 +28,9 @@ export function getActivityTileTitle(activity: Activity): string {
     } else if (activity.type === ActivityType.Consolidation) {
         return isConfirmed ? 'general.consolidated' : 'general.consolidating'
     } else if (action === ActivityAction.Mint) {
```

Comment:
I think this is correct, but literals should also be added for when an account output is `ActivityAction.Send` too

Reply:
i dont say anything 🤐